cob_bringup/tools/teleop_twist_joy.launch

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<launch>
+
+	<arg name="robot" default="$(optenv ROBOT !!NO_ROBOT_SET!!)"/>
+	<arg name="pkg_hardware_config" default="$(find cob_hardware_config)"/>
+	<arg name="sim" default="false"/>
+
+	<include file="$(find cob_bringup)/tools/teleop_common.xml" pass_all_args="true"/>
+
+	<node ns="teleop" pkg="teleop_twist_joy" type="teleop_node" name="$(anon teleop_node)" cwd="node" respawn="false" output="screen">
```

Comment:
what does this node do?

Reply:
it replaces cob_teleop since the cob version imply a load of dependencies not available on melodic (cob_script_server which relies on cob_light...)
since raw4 are melodic robots we use the ros provided, released teleop node